.pre-commit-config.yaml

```diff
@@ -1,11 +1,8 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
-# This should be the _latest_ version of python supported by us
-default_language_version:
-  python: python3.9
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v4.4.0
```

Comment:
Why the change?

Reply:
Ah, no need actually. The hooks we use haven't been updated between `3.2.0` and `4.4.0`. Reverting.